dump-dvd-kit.py

```diff
@@ -44,6 +49,7 @@
       f.write(data)
       remaining -= chunkSize
       cursor += chunkSize
+    print("Size: "+ str(code_length) + " bytes(written to dvd-dongle-rom.bin)")
```

Comment:
Please add space between "bytes" and "(written".

Maybe just  " bytes (dvd-dongle-rom.bin)".

To avoid issues in the future, we might also want to have a temporary variable for the filename (as it's used more than once, and might be changed eventually).